frontend/components/ActivityPool.vue

```diff
@@ -91,12 +105,101 @@ function showInfo(id: string) {
         }
     });
 }
+
+const confirmDelete = (event: Event) => {
+    confirm.require({
+        target: event.currentTarget as HTMLElement,
+        header: t.value("activity.delete.header"),
+        message: t.value("activity.delete.confirm"),
+        icon: "pi pi-exclamation-triangle",
+        rejectClass: "hover:underline",
+        acceptClass:
+            "text-error dark:text-error-dark hover:underline font-bold",
+        rejectLabel: t.value("common.button.cancel"),
+        acceptLabel: t.value("journey.delete"),
+        accept: () => {
+            toast.add({
+                severity: "info",
+                summary: t.value("common.toast.info.heading"),
+                detail: t.value("delete.journey.toast.message"),
+                life: 3000,
+            });
+            deleteActivity();
+        },
+    });
+};
+
+async function deleteActivity() {
+    await client(`/api/journey/${props.id}/activity/${activityId.value}`, {
+        method: "delete",
+        body: {},
+        async onResponse({ response }) {
+            if (response.ok) {
+                toast.add({
```

Comment:
Auch da weiß ich nicht von wo das kommt
Ich komm nicht mehr direkt zum Dashboard zurück. Nachdem ich den Dashboard button klicke in der JSS (Journey specific site) dann komme ich auf eine weiße seite und muss diese dann reloaden damit meine Reisen angezeigt werden.
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/a8d38800-727b-409f-ab6a-911c44e9b131)